config/CMakeAddFortranSubdirectory.cmake

```diff
@@ -130,13 +130,26 @@ with the gfortran option." )
     @ONLY)
 
   # Generate the build_cafs_proj.cmake command file:
+  include(ProcessorCount)
+  ProcessorCount(NumProcs)
+  set(build_command_args  "--build . -j ${NumProcs}")
+  if( NOT ARGS_NO_EXTERNAL_INSTALL )
+    string(APPEND build_command_args " --target install")
+  endif()
   set( build_cafs_proj_command
-"set(ENV{PATH} \"${CAFS_Fortran_COMPILER_PATH}\;\$ENV{PATH}\")
+
+"# This file generated by CMakeAddFortranSubdirectory.cmake
+#------------------------------------------------------------------------
+set(ENV{PATH} \"${CAFS_Fortran_COMPILER_PATH}\;\$ENV{PATH}\")
 set(VERBOSE ${ARGS_VERBOSE})
 if( VERBOSE )
-    message(\"${CMAKE_COMMAND} --build .\")
+    message(\"${CMAKE_COMMAND} ${build_command_args}\")
 endif()
-execute_process( COMMAND \"${CMAKE_COMMAND}\" --build . )")
+execute_process( COMMAND \"${CMAKE_COMMAND}\" ${build_command_args} )
```

Comment:
Do the build command args not need `--config` options (e.g. `--config Debug`) here?

Reply:
`${build_command_args}` will either be
* `--config Debug -j [N]`, or
* `--config Debug -j [N] --target install`